newsroom/push.py

```diff
@@ -76,7 +76,7 @@ def push():
         notify_new_item(agenda, check_topics=True)
     elif item.get('type') == 'text':
         orig = app.data.find_one('wire_search', req=None, _id=item['guid'])
-        item['_id'] = publish_item(item)
+        item['_id'] = publish_item(item, is_new=orig is None)
```

Comment:
we should make that change on master and merge it here

Reply:
Not this PR depends on https://github.com/superdesk/newsroom/pull/745